serokell-style.md

```diff
@@ -623,7 +678,20 @@ Allowed ghc forbidding option: `-fno-warn-orphans`
 
 ### Default extensions
 
+The following list of extensions is allowed to be specified inside
```

Comment:
It's better to state explicitly which are _not_ allowed to be default (because they slow down compilation or make things more complex).

Reply:
Looks more like several-days task to research which extensions can slowdown compilation (and a topic for separate blog post).

Reply:
Ok, nvm then.